### DISCUSSION TITLE
can't run "chia plotnft ..."

### DISCUSSION THREAD

**Discussion by WhyNot-hub** [Chia-Network/chia-blockchain#7037] [Support]
Hi, im trying to start chia pool, and when im trying to start comand:

`chia plotnft create -s pool -u `http://127.0.0.1:80`

**i have this message:**
 
Usage: chia [OPTIONS] COMMAND [ARGS]...
Try 'chia -h' for help.

Error: No such command 'plotnft'.

**Comment by loppefaaret** [comment]
it seems like you a trying to use developer commands, on a release version. make sure you are using a developer version if you wish to test

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
